docs/logs/settings.md

```diff
@@ -0,0 +1,84 @@
+
+  ![Sematext Logs App Settings](https://sematext.com/docs/images/logs/logsene-app-settings.png)
+
+
+## Side Navigation
+
+The persistent region on the left that can be collapsed, is used to easily switch across monitoring, log management, transaction tracing, real user monitoring, and other user & team features. All side navigation tabs are selectable. If they have chevrons, it indicates that the top level section is collapsible and contains additional subitems. The subitems will let you explore all your Apps, or choose a particular App.
+
+Side navigation top level sections for infrastructure and application performance monitoring, log management, transaction tracing, and real user monitoring:
+
+- Dashboards
+- Infrastructure
+- Monitoring
+- Logs
+- Correlations
+- Alerts and Events
+- Integrations
+
+Side navigation sections for user and team features:
+
+- Invites
+- Team
+- Account 
+
+![](../images/guide/logs/side-navigation.png)
+
+## App & Dashboard Selector
+
+The App & Dashboard dropdown selector lets you **choose any Logs or Monitoring App** as well as any **Dashboards and Reports** that you have generated from those Apps. Infrastructure views such as AppMap, NetMap, Servers, and Containers are also available for selection, and together provide seamless switching and navigation between logs, application and infrastructure monitoring, which is essential when rushing to fix production issues.
+
+## App Actions
+
+Right next to the App & Dashboard Selector is the **App Actions dropdown menu** and **horizontal ellipsis icon**. 
+
+The App Actions dropdown menu allows you to **quickly switch between your Dashboards, Logs and Monitoring Apps**. While the horizontal elipsis icon shows your App options and lets you open various app specific functions, like instructions to **install monitors and start sending metrics, app settings, alert rules, heartbeat alerts, as well as ability to connect two different apps, invite team members, and transfer app ownership**.
+
+All App Actions open in a modal dialogue window as temporary UI regions and let you adjust and edit settings or create various alerts. The Logs App view remains unchanged and allows for easy interaction between various system settings and reports views.
+
+## Report Selector
+
+Once you create a Logs App, and start shipping data, you will get a **default Report created for you named Explore**. It will have the default data and default [component](#components) configuration. You can edit this Report, clone it, or create new ones. 
+
+![](../images/guide/logs/report-selector.png)
+
+
+## Time Picker
+
+The Time Picker is available whenever a Logs or Monitoring App is selected as well as when any custom dashboards are being used. **One minute is the shortest time increment** you can select. **Thirty minutes, one and two hours, and one, two, seven, and thirty days time-span shortcuts** are exposed common observability defaults.
```

Comment:
Generalize?
Plus I think time picker is displayed on views where a single App is not selected, too.
"are exposed common observability defaults" - something sounds strange here.

Reply:
Fixed wording and added generalization.